scripts/buildWebsite.js

```diff
@@ -135,6 +136,7 @@ async function build() {
   fs.copySync(path.join(__dirname, '..', 'starters'), path.join(dir, 'starters'));
   fs.copySync(path.join(__dirname, '..', '.yarn', 'releases'), path.join(dir, '.yarn', 'releases'));
   fs.copySync(path.join(__dirname, '..', '.yarn', 'plugins'), path.join(dir, '.yarn', 'plugins'));
+  fs.copySync(path.join(__dirname, '..', '.yarn', 'patches'), path.join(dir, '.yarn', 'patches'));
```

Comment:
could probably do what we do for buildBranchAPI and just copy everything in `.yarn` instead of picking and choosing